### ISSUE TITLE
SDK Scheme Adapter is not accepting PUT /parties with 1.1 content-type header

### ISSUE THREAD

**Issue by vijayg10** [mojaloop/project#1891]
**Summary**:
SDK Scheme Adapter is not accepting **PUT /parties** with **1.1 content-type header**
But post /quotes and post /transfers are working fine with 1.1 headers

**Severity**:
(Low, Medium, High)

**Priority**:
(Critical, Medium, Low)

**Expected Behavior**
SDK should accept get /parties with content-type header 1.1.

**Steps to Reproduce**
  1. Send a PUT request with content-type header **"application/vnd.interoperability.parties+json;version=1.1"**
  2. We will get 400 http error code with following body
       ```
        {
            "errorInformation": {
                "errorCode": "3102",
                "errorDescription": "undefined must have required property 'body'"
            }
        }
       ```
  3. Send a PUT request with content-type header **"application/vnd.interoperability.parties+json;version=1.0"**
  4. We will get 200

**Specifications**
  - Component (if known): sdk-scheme-adapter
  - Version:
  - Platform:
  - Subsystem:
  - Type of testing:
  - Bug found/raised by: @vijayg10 

**Notes**:

- Severity when opened:
- Priority when opened:

**Comment by elnyry-sam-k** [comment]
@vijayg10 , @shashi165 - can you please confirm if this is still relevant / active?

**Comment by vijayg10** [comment]
@elnyry-sam-k I check it just now with master branch, issue still exists

**Comment by elnyry-sam-k** [comment]
Thanks Vijay

**Comment by mdebarros** [comment]
This has been fixed on [#275](https://github.com/mojaloop/sdk-scheme-adapter/pull/275) PR - https://github.com/mojaloop/sdk-scheme-adapter/pull/275/commits/039c06d1f6b2316882349789634d65739f386819

This will be included in [Helm v12.1.0 release](https://github.com/mojaloop/project/issues/2151).

There is a follow up story that I created based on my analysis of the SDK Adapter's current functionality: https://github.com/mojaloop/project/issues/2264

Specifically that the Adapter DOES NOT validate the incoming content-type header, and does not match the correct "media-type" to the FSPIOP operation.

**Comment by elnyry-sam-k** [comment]
Fixed with https://github.com/mojaloop/helm/releases/tag/v13.0.0
